app/frontend/actions/wallet.ts

```diff
@@ -60,29 +60,38 @@ export default (store: Store) => {
     {
       cryptoProviderType,
       walletSecretDef,
-      forceWebUsb,
+      ledgerTransportType,
       shouldExportPubKeyBulk,
     }: {
       cryptoProviderType: CryptoProviderType
       walletSecretDef?: any // TODO: until now, arguments came in freestyle combinations, refactor
-      forceWebUsb?: boolean // into strict combinations and types
+      ledgerTransportType?: LedgerTransportType
       shouldExportPubKeyBulk: boolean
     }
   ) => {
     loadingAction(state, 'Loading wallet data...')
     setState({walletLoadingError: undefined})
     const isShelleyCompatible = !(walletSecretDef && walletSecretDef.derivationScheme.type === 'v1')
-    const config = {...ADALITE_CONFIG, isShelleyCompatible, shouldExportPubKeyBulk}
+    const config = {
+      ...ADALITE_CONFIG,
+      isShelleyCompatible,
+      shouldExportPubKeyBulk,
+      ledgerTransportType,
+    }
     try {
+      loadingAction(
+        state,
+        'Please allow access to your device. Select your device in dropdown and then click Connect.'
```

Comment:
this message doesn't apply for U2F where there isn't any such dropdown

Reply:
I'd consider rewording it/reformatting it as a two-line text:

```
Loading wallet data...`
If a prompt appears, click on the Ledger device, then click "Connect."
```
The instruction `Please allow access to your device.` is imho too ambiguous and in some instances the users would not need to confirm anything but the public key on the device, contributing to further confusion

Reply:
Fixed